spec/dummy/config/alchemy/elements.yml

```diff
@@ -83,32 +83,41 @@
   contents:
     - name: essence_text
       type: EssenceText
+      editor_css_classes: essence_text_css_classes
```

Comment:
Please remove this. This is not the right place and we should just use the above mentioned `data-` attributes for that.